fees/avalon-finance-cedefi/index.ts

```diff
@@ -0,0 +1,44 @@
+import { Adapter, FetchV2 } from '../../adapters/types'
+import { CHAIN } from '../../helpers/chains'
+
+const abi = {
+	getPoolManagerReserveInformation:
+		'function getPoolManagerReserveInformation() view returns (tuple(uint256 userAmount, uint256 collateral, uint256 debt, uint256 claimableUSDT, uint256 claimableBTC) poolManagerReserveInfor)',
+	getPoolManagerConfig:
+		'function getPoolManagerConfig() view returns (tuple(uint256 DEFAULT_LTV, uint256 DEFAULT_LIQUIDATION_THRESHOLD, uint256 DEFAULT_POOL_INTEREST_RATE, uint256 DEFAULT_PROTOCOL_INTEREST_RATE, address USDT, address FBTC0, address FBTC1, address FBTCOracle, address AvalonUSDTVault, address AntaphaUSDTVault))',
+}
+
+const poolAddress = '0xF95308B15E45BFF3d0811B5732d17Eb2A555afB2'
```

Comment:
this is not a valid contract?

Reply:
I have pushed a change where the contract is changed to one used in tvl repo, please fix it if it is wrong